client/src/main/java/com/aws/greengrass/cli/commands/Logs.java

```diff
@@ -92,18 +92,17 @@ public int get(@CommandLine.Option(names = {"-lf", "--log-file"}, paramLabel = "
             return null;
         }
         List<Path> paths = new ArrayList<>();
-        for (int i = 0; i < arr.length; i++) {
-            String s = arr[i];
+        for (String s : arr) {
             paths.add(deTilde(s).orElse(Paths.get("")));
         }
         return paths;
     }
 
     @Command(name = "list-log-files", mixinStandardHelpOptions = true,
             versionProvider = com.aws.greengrass.cli.module.VersionProvider.class)
-    public void listLogFiles(@CommandLine.Option(names = {"-ld", "--log-dir"}, paramLabel = "Log directory")
-                                 String[] logDir) {
-        List<Path> logDirList = Arrays.stream(logDir).map(d -> Paths.get(d)).collect(Collectors.toList());
+    public void listLogFiles(@CommandLine.Option(names = {"-ld", "--log-dir"}, paramLabel = "Log directory",
+            required = true) String[] logDir) {
```

Comment:
Should we default to ggcRoot/logs if not present? documentation says we do that

Reply:
Where do we say that? We don't do that anywhere for any of the logs commands